specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -10912,6 +10915,26 @@
       },
       "title": "The execution constraints for a job."
     },
+    "TaskContainerSettings": {
+      "properties": {
+        "containerRunOptions": {
+          "type": "string",
+          "title": "Additional options to the container run command.",
+          "description": "These additional options are supplied as arguments to the \"docker create\" command, in addition to those controlled by the Batch Service."
+        },
+        "imageName": {
+          "type": "string",
+          "title": "The container image under which the task will run."
+        },
+        "registry": {
+          "$ref": "#/definitions/ContainerRegistry",
+          "title": "Details about the private registry which contains the container image.",
```

Comment:
Is "Details about" doing useful work here?

Reply:
No -- fixed.